packages/headless/src/api/search/facet-search/category-facet-search/category-facet-search-request.test.ts

```diff
@@ -23,77 +23,78 @@ describe('#buildCategoryFacetSearchRequest', () => {
 
   beforeEach(() => setupState());
 
-  it('retrieves the #captions from the categoryFacetSearchSet', () => {
+  it('retrieves the #captions from the categoryFacetSearchSet', async () => {
```

Comment:
fyi, I moved the facet search test files next to the files with the builder functions they test,